src/main/resources/openapi.json

```diff
@@ -1564,7 +1625,129 @@
                         }
                     ]
                 }
-            }
+            },
+            "TopicMetadata": {
+                "title": "TopicMetadata",
+                "type": "object",
+                "properties": {
+                    "name": {
+                        "description": "Name of the topic",
+                        "type": "string"
+                    },
+                    "configs": {
```

Comment:
it seems to be not retrieved from the code.

Reply:
added the configs